frontend/src/data/variables/CdcCovidProvider.ts

```diff
@@ -116,6 +116,27 @@ class CdcCovidProvider extends VariableProvider {
       })
       .resetIndex();
 
+    // Drop unused columns for simplicity.
+    df = df.dropSeries(["death_n", "death_unknown", "hosp_n", "hosp_unknown"]);
+
+    // Clear all county-level DC data. See issue for more details:
+    // https://github.com/SatcherInstitute/health-equity-tracker/issues/872.
+    // TODO - fix this the right way.
+    df = df.withSeries({
```

Comment:
is it worth extracting into a util so it's easy to reuse if needed in the future?

Reply:
Hmm, my feeling is that's not needed for now, and we can extract into a util in the future as needed